### ISSUE TITLE
Измени шрифт

### ISSUE THREAD

**Issue by igorlukanin** [Tank090493/Test-pilot#5]
Многие сайты используют не только шрифты, которые есть на компьютере пользователя, но и «приносят шрифты с собой». Один из самых популярных способов — взять шрифт в бесплатном каталоге шрифтов от Гугла: https://fonts.google.com

Выбери шрифт без засечек по душе (например, Open Sans, Roboto, Lato), выбери его параметры (чтобы не загружать вместе со страницей лишние начертания и символы для редких языков), и с помощью CSS сделай так, чтобы весь текст на твоей странице рисовался этим шрифтом.

С помощью веб-инспектора в браузере определи, насколько увеличился общий размер твоей страницы и время её загрузки. Приложи сюда скриншот, подтверждающий эти цифры.

**Comment by Tank090493** [comment]
Сделано.

**Comment by Tank090493** [comment]
![default](https://user-images.githubusercontent.com/27670568/27636120-6699a402-5c23-11e7-976a-6c49c5afd54d.png)

**Comment by igorlukanin** [comment]
1. Я просил выбрать параметры шрифта, чтобы не загружать лишние начертания. Ты [загружаешь](https://github.com/Tank090493/Test-pilot/commit/0759a4dd5625f89bf5cd82d6302fd0b6b3d5756c#diff-eacf331f0ffc35d4b482f1d15a887d3bR4) 4 начертания: обычное (400), обычное курсивное (400i), жирное (700) и жирное курсивное (700i). При этом на странице нет ни символа, набранного курсивом :) Давай-ка и загружать их не будем, чтобы просто так не увеличивать размер страницы.

2. Вижу скриншот, не вижу ответа на вопрос «насколько увеличился общий размер твоей страницы и время её загрузки». Ты сама-то видишь ответ на этот вопрос на скриншоте? :) Напиши цифрами, сколько было «до» и сколько стало «после». И подтверди это скриншотом.

**Comment by Tank090493** [comment]
Так, тут я тоже все исправила, получается?)) Только скриншот вопрос, да?)

**Comment by igorlukanin** [comment]
Да, пункт 2 остался.

**Comment by Tank090493** [comment]
![default](https://user-images.githubusercontent.com/27670568/34072171-ef732380-e29b-11e7-9927-e3c1d5f645a9.png)
Размер страницы.
Было: 373В. Стало: 5.8КВ.
Время загрузки.
Было: 195ms. Стало: 24ms
